### ISSUE TITLE
Mention "require" in README?

### ISSUE THREAD

**Issue by mcary** [flavorjones/chromedriver-helper#66]
I've been having some issues that might be related to PR #58, which changes the `chromedriver` executable file name to `chromedriver-helper`:
```
  unable to connect to chromedriver 127.0.0.1:9515 (Selenium::WebDriver::Error::WebDriverError)
```
I'm using this outside of Rails in a standalone test suite to test an app that is developed separately; not sure if that has an impact.  So my Gemfile contains:
```
# no rails
gem "cucumber"
gem "rspec-expectations"
gem "pry"
gem "capybara"
gem "selenium-webdriver"
gem "chromedriver-helper"
```
I found that Selenium::WebDriver::Chrome.driver_path was nil after trying to visit a page (after rescuing from the failure).

The problem was that I was never loading the new chromedriver-helper code to set the custom driver_path.  I added this to features/support/env.rb and my scenario started working:
```
require 'chromedriver-helper'
```
Perhaps the README could mention this in the [Standalone](https://github.com/flavorjones/chromedriver-helper#standalone) section?   I did look there for how to install it.

I'm happy to submit a doc PR if this sounds helpful.

**Comment by flavorjones** [comment]
Sorry, this is a bit of a mess after that PR. Really sorry. Will take a deeper look at this later tonight.

**Comment by ioquatix** [comment]
I ran into this problem too. It broke our tests. It's non-obvious, I think you should restore the original behaviour.

**Comment by flavorjones** [comment]
I've merged #67 which updates the docs to mention this. Apologies for the long turnaround time.
